### DISCUSSION TITLE
The Best Experience in Remote Talks that I Have Given | /en/2018/06/best-remote-talk/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1623] [Comments]
This morning I gave a remote talk to a summer school in Sweden named &ldquo;RaukR&rdquo;. It is the first time I have felt completely satisfied by the video conference equipment. The audio quality was &hellip;

https://yihui.org/en/2018/06/best-remote-talk/

**Comment by think-crow** [comment]
原来英文文章也这么有料，看不懂英文的我一直以为是中文博客的翻译版，误会太深太深！
第一次了解hugo中英文切换机制的时候还在想：这么追求极简的一个人，怎么会一篇文章写两遍。

想体验一下这个高质量的音频，发现链接404了：https://www.scilifelab.se/events/raukr-advanced-r-for-bioinformatics-summer-school/

**Reply by yihui** [reply]
这个报告没有录下来。

链接看样子是找不回来了，谢谢报告。
